read-fonts/build.rs

```diff
@@ -9,6 +9,7 @@
 use std::path::Path;
 
 static FILES_TO_MOVE: &[&str] = &[
+    "../resources/test_fonts/ttf/cmap14_font1.ttf",
```

Comment:
This reminds me, it's fairly easy to forget this and find out when the file isn't available and you try to use it. Should we just copy ttf/*.ttf?

Reply:
We should and it would also be nice to generate the file that includes the font data as well. I'd say this belongs in another PR though.